etl/steps/data/garden/energy/2023-12-12/photovoltaic_cost_and_capacity.meta.yml

```diff
@@ -1,43 +1,36 @@
-
-    Prices from Farmer & Lafond (2016) have been converted to 2021 US$ using the US GDP deflator: https://www.multpl.com/gdp-deflator/table/by-year
-
-    Photovoltaic capacity data between 2004 and 2021 has been taken from IRENA.
-
-    Photovoltaic cost data between 2010 and 2021 has been taken from IRENA.
 tables:
   photovoltaic_cost_and_capacity:
+    title: Solar photovoltaic cost and capacity
     variables:
       cost:
         title: Solar photovoltaic module price
         short_unit: $/W
         unit: 2021 US$ per Watt
-        description: |
-          Global average price of solar photovoltaic modules.
-
-          IRENA presents solar PV module price series for a number of different module technologies. Here we have adopted the series for thin film a-Si/u-Si or Global Index (from Q4 2013).
+        description_key:
+          - IRENA presents solar PV module price series for a number of different module technologies. Here we use the series for thin film a-Si/u-Si or Global Index (from Q4 2013).
       cost_source:
-        title: Data source for cost data
+        title: Source for each value of cost data
         unit: ''
-        description: Source for each value of cost data.
       cumulative_capacity:
         title: Solar photovoltaic cumulative capacity
```

Comment:
again you are providing v little info on what this metric actually is - does it represent the maximum potential amount of electricity that can be generated by all solar PV installations under ideal conditions?